docs/Glossary.md

```diff
@@ -0,0 +1,172 @@
+# Definitions
+
+There is seperate section [terminology](./spec/terminology.md) is a separate file additional to this Glossary for the Q&A. The terminology is for advanced to proficient Sidetree users.
+
+## Abbreviations
+In alphabetic order:\
+ACDC = Authentic Chained Data Container Task Force
+BX = [Bidirectional model transformation](#bidirectional-model-transformation)
+CAS = Content Addressable Storage 
+DID = [Decentralized Identity](#decentralized-identity) or Digital Identity dependent of the context.\
+DIF = Decentralized Identity Foundation\
+DDO = DID Document, look up W3D DID standardization for more info\
+DHT = Distributed Hash Table\
+DIF = Decentralized Identity Foundation, https://identity.foundation\
+DKMI = Decentralized Key Mangement Infrastructure\
+JWK = [JSON Web Key](#json-web-key)
+IPv4 = standard Internet Protocol, version 4\
+PKI = [Public Key Infrastructure](#public-key-infrastructure)\
+PR = Pull Request; github terminology\
+SDK = Software Development Kit\
+SSI = [Self Sovereign Identity](#self-sovereign-identity)\
+VC = Verifiable Credential, look up W3C Verifiable Credentials for more info\
+WASM = [WebAssembly](#WebAssembly)
+
+## Definitions in alphabetic order
+
+Terminology specific for Sidetree can be found [here](https://identity.foundation/sidetree/spec/#terminology). The following definitions are specificly listed to support the [Question and Answers](Q-and-A.md) on a simpler level, and to be able to put Sidetree in relation to other fields of interest.
+
+
+#### Authentic Chained Data Container Task Force
```

Comment:
Why are we referencing a ToIP task force in the Sidetree Q & A?

Reply:
Feel free to cut it out. I am not aware of sensitivities.